src/components/AccountDrawer/UniwalletModal.tsx

```diff
@@ -51,21 +48,12 @@ export default function UniwalletModal() {
     uniswapWalletConnectors.includes(activationState.connection.type) &&
     !!uri
 
-  const walletConnectV2AsDefault = useWalletConnectV2AsDefault()
-
   useEffect(() => {
-    if (walletConnectV2AsDefault) {
-      const connectorV2 = uniwalletWCV2ConnectConnection.connector as WalletConnectv2
-      connectorV2.events.addListener(UniwalletConnectV2.UNI_URI_AVAILABLE, (uri: string) => {
-        uri && setUri(uri)
-      })
-    } else {
-      const connectorV1 = uniwalletConnectConnection.connector as WalletConnect
-      connectorV1.events.addListener(UniwalletConnect.UNI_URI_AVAILABLE, (uri: string) => {
-        uri && setUri(uri)
-      })
-    }
-  }, [walletConnectV2AsDefault])
+    const connectorV2 = uniwalletWCV2ConnectConnection.connector as WalletConnectv2
+    connectorV2.events.addListener(UniwalletConnectV2.UNI_URI_AVAILABLE, (uri: string) => {
+      uri && setUri(uri)
+    })
+  }, [])
```

Comment:
nit: do we still want this wrapped in a useEffect if there's no depedencies?

Reply:
i think this is async so it has to be in an effect?